tensorflow_datasets/core/download/downloader_test.py

```diff
@@ -128,8 +129,21 @@ def test_bad_http_status(self):
   def test_kaggle_api(self):
     fname = 'a.csv'
     with testing.mock_kaggle_api(filenames=[fname, 'b.txt']):
-      promise = self.downloader.download('kaggle://some-competition/a.csv',
-                                         self.tmp_dir)
+      # Testing Competition Downloader
+      promise = self.downloader.download(
+          'kaggle://%s/some-competition/a.csv' % \
+          (kaggle.KaggleFile._COMPETITION_PREFIX),
```

Comment:
I guess both of them are same, since `kaggle.KaggleFile._COMPETITION_PREFIX` contains `competition` as a value. I think using the variable instead of hardcoding the prefix, needs rewriting this section of the test, incase the prefix needs some patch in future. What do you think?